modules/engage-paella-player-7/src/js/EpisodeConversor.js

```diff
@@ -347,37 +361,6 @@ function getCaptions(episode) {
   // Read the tracks
   readCaptions(tracks, captions);
 
-  // Read the catalogs
```

Comment:
There are some institutions using subtitles from catalogs. See (https://github.com/opencast/opencast/pull/5049#issuecomment-1599126063).

I believe that we should not remove this support until a way to transfer these subtitles to tracks is provided. Or at least add a note that this feature will be removed in future versions to give the institutions time to upgrade.

Perhaps we could add a configuration variable in the config.json: `deprecatedCaptionsAsCatalog: false`. By default it could be set to false and if any institution needs it, they can activate it and realize that it is a deprecated feature and in future versions it will be removed.

@karendolan?

Reply:
For release notes and migrations notes see this PR: https://github.com/opencast/opencast/pull/4844 I've split the subtitle-rework into multiple PRs to make them easier to work on and hopefully also easier to review.

If enough people are in favor of it, I don't mind keeping the existing "captions as catalog" functionality and marking it as deprecated.

Reply:
@Arnei. Is it possible to use the `changetype` operation to change the type of subtitles from catalogs to tracks (from PR: #4844)?

Reply:
That's the intention, yes.

Reply:
Release notes from PR #4844 are ok for me. 
If nobody complains about removing the subtitles from the catalogs in this release, it's fine with me (there is an operation so that the type can be changed).

The support of subtitles in the catalogs has been removed, but the support of subtitles in the attachments is kept. Why is it not removed as well?

https://github.com/opencast/opencast/blob/0de9f6118f55390169fbdfa7b5d62ef41aa9bcb4/modules/engage-paella-player-7/src/js/EpisodeConversor.js#L358-L362